BST.cs

```diff
@@ -0,0 +1,180 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BST
+{
+    class BST
+    {
+
+        public Node root;
+
+
+        public BST()
+        {
+            root = null;
+        }
+
+        public void insert(int key)

```

Comment:
Method name notation should be PascalCase. Useful link:
https://github.com/ktaranov/naming-convention/blob/master/C%23%20Coding%20Standards%20and%20Naming%20Conventions.md